### ISSUE TITLE
Add support for computed fields

### ISSUE THREAD

**Issue by yuvalek** [kvesteri/sqlalchemy-continuum#340]
The code today does not support computed fields.
When sqlalchemy-continuum tried to create the versioned row it also tried to write a value to the computed column thus making an error.

**Comment by yuvalek** [comment]
My pr excludes computed columns when writing but still creates them in the DB.

**Comment by yoshpe** [comment]
I've tested it, however the function `versioned_column_properties` isn't seemed to be called when generating the Metadata schema (the columns are added to the history's table)

As a workaround, you can add 'exclude' to the `__versioned__` key, and manually exclude the column, which works

**Comment by marksteward** [comment]
Yeah I'm not sure how we'd do this any way other than ignoring them. Suggestions welcome.

**Comment by yuvalek** [comment]
It is weird because I tested it locally when I wrote it and it did work and excluded the column.
